launcher/launcher/main.go

```diff
@@ -79,17 +79,21 @@ func main() {
 		return
 	}
 
-	experimentsFile := path.Join(launcherfile.HostTmpPath, experimentDataFile)
+	err = os.MkdirAll(launcherfile.HostExperimentsPath, os.ModePerm)
```

Comment:
I think your error is that you don't create the directory first. It's created in https://github.com/google/go-tpm-tools/blob/master/launcher/container_runner.go#L404.
So pull that mkdir here.

Reply:
Done